### ISSUE TITLE
[Android] Touches below screen's center will close the modal, regardless of the position

### ISSUE THREAD

**Issue by Sh3rawi** [maxs15/react-native-modalbox#18]
Hello,
Great work on the modal (Y).
Im testing this on android. Here is a gif that explains everything.
Im using react-native 0.14

![modalbox_issue](https://cloud.githubusercontent.com/assets/13085991/11004363/73b8c8e8-84c0-11e5-8528-10e7269ca7d8.gif)

**Comment by Sh3rawi** [comment]
I tried to overcome this by setting isDisabled to true after i open the modal and then to set it to false just before i want to close the modal. But the problem that arised is that i can't press anything (<TouchableOpacity>) unless i am in the top position. If im centered or at the bottom the click is not recognized at all(no opacity change or anything).

**Comment by maxs15** [comment]
Hey @Sh3rawi, thanks for the bug report !
I will look at that

**Comment by maxs15** [comment]
Everything seems to work on my side, did you try the example ?

**Comment by Sh3rawi** [comment]
Yes i tried it, on react-native 0.13 and 0.14, on simulator and physical device, but still the same problem occurs.

**Comment by kickboxer** [comment]
I have the same problem
